### DISCUSSION TITLE
Do we have a scrollbar component in a code block?

### DISCUSSION THREAD

**Discussion by charleneyjh1** [twilio-labs/paste#1857] [Q&A] [answered]
Hi team,

I wonder do we have a horizontal scroll bar component that we can use in a code block? (See design below)

If no, do you have any recommendation on how to approach this?

Thank you in advance!

<img width="588" alt="Screen Shot 2021-09-10 at 12 34 31 PM" src="https://user-images.githubusercontent.com/56697620/132901366-e51645da-299f-49c3-8083-b0011968466b.png">

**Comment by nkrantz** [comment, marked as answer]
Hi @charleneyjh1,

We don't currently have a Paste Code Block component (it's on the [roadmap](https://paste.twilio.design/roadmap), release date TBD), but you should be able to accomplish the horizontal scroll with whatever approach you're currently taking for the code block. Adding the css property `overflow-x="scroll"` should take care of it.

If you'd like more support, feel free to leave a code sandbox link and we will take a look.

**Comment by charleneyjh1** [comment]
Thanks Nora! I'll let my engineer know.
